ccd-definition/CaseEventToFields/draftCMO.json

```diff
@@ -10,5 +10,41 @@
     "PageLabel": " ",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "orderBasis",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "2",
```

Comment:
pageID doesn't need to be a number. In draftSDO I tried to make it things that made sense for what the page is. Recitals might be a more meaningful ID?

Reply:
ah cool didn't realise this, that makes more sense to have it more meaningful